### DISCUSSION TITLE
Snaps Discovery

### DISCUSSION THREAD

**Discussion by vinny023** [MetaMask/snaps#932] [Q&A] [answered]
Hi everyone! Super excited about Snaps and pumped to buidling on it.

One question - how will the discovery of snaps work for metamask users? Will there an app store type directory somewhere?

**Comment by ritave** [comment, marked as answer]
Hey @vinny023,

Currently MetaMask doesn't plan to create an app store, we would like the community to self organise.

There are already such 3rd party initiatives running, such as [awesome-metamask-snaps](https://github.com/piotr-roslaniec/awesome-metamask-snaps), run by @piotr-roslaniec.
